docs/resources/csms_secret.md

```diff
@@ -0,0 +1,63 @@
+---
+subcategory: "Data Encryption Workshop (DEW)"
+---
+
+# huaweicloud_csms_secret
+
+Manages CSMS(Cloud Secret Management Service) secrets within HuaweiCloud.
+
+## Example Usage
```

Comment:
suggest to add an example of `secret_text` in JSON format as same as the console.

Reply:
ok